crates/storage/error.rs

```diff
@@ -52,4 +52,6 @@ pub enum StoreError {
     MempoolReadLock(String),
     #[error("Failed to lock database for writing")]
     LockError,
+    #[error("Incompatible chain configuration")]
+    IncompatibleChainConfig(),
```

Comment:
The parentheses are not needed

```suggestion
    IncompatibleChainConfig,
```